setup.cfg

```diff
@@ -9,7 +9,13 @@ max-line-length = 120
 
 [flake8]
 max-line-length = 120
-ignore = F401,E402,F403,W503,W504,F821
+ignore = E203,E402,F403,W503,W504,F821
+per-file-ignores =
```

Comment:
What is the reason for this change, in particular switching to ignoring F401 on a per-file basis?

Reply:
F401 is unused imports. They were bugging me for some time. In some modules like `__init__.py` though we simply import stuff for accessibility. In these files we simply ignore the unused imports.

Reply:
E203 is whitespace around slice syntax. `black` and `flake8` disagree on this and thus we ignore it to be able to use `black`.

Reply:
Thanks for the details, sounds good. I feel like we should clean up those `import *` eventually... but let's leave it for future PRs, might be a good-first-issue thing